torch/linalg/__init__.py

```diff
@@ -472,7 +473,7 @@
     ``'gelsy'`` is the fastest among the rank-revealing algorithms that also handles rank-deficient inputs.
 
 .. warning::
-    The default value for :attr:`cond` is subject to a potential change.
+    The default value for :attr:`rcond` is subject to a potential change.
```

Comment:
"... is subject to a potential change" -> "... may change in a future PyTorch release."